packages/fast-usdc/src/exos/operator-kit.js

```diff
@@ -36,19 +42,21 @@ const OperatorKitI = {
 
 /**
  * @param {Zone} zone
- * @param {{handleEvidence: Function, makeInertInvitation: Function}} powers
+ * @param {{ makeInertInvitation: Function }} staticPowers
  */
-export const prepareOperatorKit = (zone, powers) =>
+export const prepareOperatorKit = (zone, staticPowers) =>
   zone.exoClassKit(
     'Operator Kit',
     OperatorKitI,
     /**
      * @param {string} operatorId
+     * @param {OperatorPowers} powers
      * @returns {State}
      */
-    operatorId => {
+    (operatorId, powers) => {
       return {
         operatorId,
+        powers,
```

Comment:
I hunted around to be sure that a durable `powers` object is getting supplied. Add `Remote` to the `OperatorPowers` type to clarify, perhaps? No... we call them synchronously. hm. Call it `OperatorFacet`? or add a comment?